heron/schedulers/src/java/com/twitter/heron/scheduler/yarn/YarnLauncher.java

```diff
@@ -153,6 +155,7 @@ private Configuration getHMDriverConf() {
         .set(HeronDriverConfiguration.HTTP_PORT, 0)
         .set(HeronDriverConfiguration.VERBOSE, false)
         .set(YarnDriverConfiguration.QUEUE, queue)
+        .set(DriverConfiguration.DRIVER_MEMORY, driverMemory)
```

Comment:
Good point. I have some ideas to test the yarnconfigs. Will create an issue to track it separately. Thanks.